.env

```diff
@@ -1 +1,2 @@
-NOTION_TOKEN=secret_rrm731Cn5OVwp9iV89ck5fgZtMh94OfXRe7IBRI1m2h
\ No newline at end of file
+NOTION_TOKEN=secret_rrm731Cn5OVwp9iV89ck5fgZtMh94OfXRe7IBRI1m2h
+GA_MEASUREMENT_ID=GA_H63MN8CMZF
```

Comment:
Just to make sure: The secret starts with `GA_`, but GA measurement IDs start with `G-`. Does `ReactGA` require it like this, i.e., with `GA_`?

Reply:
Oops my bad, I will look into that 🙈